marge/app.py

```diff
@@ -27,138 +28,153 @@ def time_interval(s):
+        description=__doc__,
+    )
+    auth_token_group = parser.add_mutually_exclusive_group(required=True)
+    auth_token_group.add_argument(
+        '--auth-token',
+        type=str,
+        metavar='TOKEN',
+        help=(
+            'Your gitlab token.\n'
+            'DISABLED because passing credentials on the command line is insecure:\n'
+            'You can still set it via ENV variable or config file, or use "--auth-token-file" flag.\n'
+        ),
+    )
+    auth_token_group.add_argument(
         '--auth-token-file',
         type=argparse.FileType('rt'),
         metavar='FILE',
-        help='Your gitlab token; must provide this flag or set MARGE_AUTH_TOKEN',
-        env_var=False,
+        help='Path to your gitlab token file.\n',
     )
-    arg(
+    parser.add_argument(
         '--gitlab-url',
         type=str,
         required=True,
         metavar='URL',
-        help='Your gitlab instance, e.g. https://gitlab.example.com',
+        help='Your gitlab instance, e.g. "https://gitlab.example.com".\n',
     )
-    arg(
-        '--ssh-key-file',
+    ssh_key_group = parser.add_mutually_exclusive_group(required=True)
+    ssh_key_group.add_argument(
+        '--ssh-key',
         type=str,
-        metavar='FILE',
+        metavar='KEY',
         help=(
-            'Path to the private ssh key for marge so it can clone/push. '
-            'Provide or set MARGE_SSH_KEY (to the *contents*)'
+            'The private ssh key for marge so it can clone/push.\n'
+            'DISABLED because passing credentials on the command line is insecure:\n'
+            'You can still set it via ENV variable or config file, or use "--ssh-key-file" flag.\n'
         ),
-        env_var=False,
     )
-    arg(
+    ssh_key_group.add_argument(
+        '--ssh-key-file',
+        type=str,  # because we want a file location, not the content
+        metavar='FILE',
+        help='Path to the private ssh key for marge so it can clone/push.\n',
+    )
+    parser.add_argument(
         '--embargo',
         type=interval.IntervalUnion.from_human,
         metavar='INTERVAL[,..]',
-        help='Time(s) during which no merging is to take place, e.g. "Friday 1pm - Monday 9am".',
+        help='Time(s) during which no merging is to take place, e.g. "Friday 1pm - Monday 9am".\n',
     )
-    arg(
+    parser.add_argument(
         '--add-tested',
         action='store_true',
-        help='add Tested: marge-bot <$PR_URL> for the final commit on branch after it passed CI',
+        help='Add "Tested: marge-bot <$MR_URL>" for the final commit on branch after it passed CI.\n',
     )
-    arg(
+    parser.add_argument(
         '--add-part-of',
         action='store_true',
-        help='add Part-of: <$PR_URL> to each commit in PR (except top commit when --add-tested is used)',
+        help=(
+            'Add "Part-of: <$MR_URL>" to each commit in MR\n'
+            '(except top commit when --add-tested is used).\n'
```

Comment:
@mrusu91 Actually, I think the `help` entry here was wrong: we are adding `Part-of: ` on every commit, regardless of whether `--add-tested` is used.

Reply:
so should are remove the 2nd line?

Reply:
yes, please!